app/models/post.rb

```diff
@@ -10,10 +10,15 @@ class Post < ApplicationRecord
 
   after_create do
     ActionCable.server.broadcast "posts_realtime", { row: PostsController.render(locals: {post: Post.last}, partial: 'post').html_safe }
+    ActionCable.server.broadcast "topbar", { review: Post.feedbacks_count }
   end
 
   after_create :autoflag
 
+  def self.feedbacks_count
+    Post.includes(:feedbacks).where( :feedbacks => { :post_id => nil }).count
```

Comment:
This doesn't do what you think it does. `Post.includes(:feedbacks)` never has `feedbacks.post_id = nil`, because it uses a (lazy) INNER JOIN. You want `Post.left_joins(:feedbacks)` here; in general, `Post.joins(:table)` is a safer bet than `includes` because of the lazy loading.